laika/raw_gnss.py

```diff
@@ -162,15 +168,17 @@ def group_measurements_by_sat(measurements):
 
 def read_raw_qcom(report):
   dr = 'DrMeasurementReport' in str(report.schema)
-  if report.source == 0 or report.source == 6:    # gps/sbas
+  constellation_id = ConstellationId.from_qcom_source(report.source)
+  # Only gps/sbas and glonass are supported
+  if constellation_id is not ConstellationId.GLONASS:  # gps/sbas
```

Comment:
Let's not change the logic in this PR and keep it like `constellation id in [ConstellationId.GPS, ConstellationId.SBAS]`.

Reply:
Will do!